src/Http/Controllers/ChannelController.php

```diff
@@ -74,6 +75,13 @@ public function show(Channel $channel)
         return view('vanilo::channel.show', $this->processViewData(__METHOD__, [
             'channel' => $channel,
             'products' => (new ProductSearch())->withinChannel($channel)->withInactiveProducts()->getResults(),
+            'country' => CountryProxy::where('id', $channel->configuration()['country_id'])->first(),
+            'orderCount' => DB::table('orders')->where('channel_id', $channel->id)->count(),
+            'ordersPerCurrency' => DB::table('orders')
```

Comment:
Same here. The DB instead of the model should only be used in migrations, seeds and other cases where we go directly with the table.